arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -95,7 +95,7 @@ public async Task PostCollectionAsync_ShouldSucceed_WhenUsingKeyOptions()
             Assert.Equal("MyCollectionWithKeyOptions", response.Name);
             Assert.Equal("autoincrement", response.KeyOptions.Type);
             Assert.False(response.KeyOptions.AllowUserKeys);
-            Assert.Equal(2, response.Type); // 2 is document collection, 3 is edge collection
+            Assert.Equal(CollectionType.Document, response.Type); // 2 is document collection, 3 is edge collection
```

Comment:
Remove the comment // 2 is document collection, 3 is edge collection?

Reply:
Thanks good spot, it's minor so might leave it to be tidied up later